tests/test_config.py

```diff
@@ -144,3 +144,7 @@ def test_initialize_tracer(self):
         tracer = c.initialize_tracer()
 
         assert opentracing.tracer == tracer
+
+    def test_default_local_agent_reporting_port(self):
+        c = Config({}, service_name='x')
+        assert c.local_agent_reporting_port == DEFAULT_REPORTING_PORT
```

Comment:
rather than testing against DEFAULT_REPORTING_PORT, can you explicitly test against the value? This way next time someone inadvertently changes DEFAULT_REPORTING_PORT to something else, this test will break.

Reply:
Updated.